src/Playwright/Contracts/IBrowserType.generated.cs

```diff
@@ -197,12 +200,12 @@ public partial interface IBrowserType
         /// <param name="ignoreHTTPSErrors">Whether to ignore HTTPS errors during navigation. Defaults to <c>false</c>.</param>
         /// <param name="bypassCSP">Toggles bypassing page's Content-Security-Policy.</param>
         /// <param name="viewportSize">
-        /// Emulates consistent viewport for each page. Defaults to an 1280x720 viewport. Use
-        /// <c>ViewportSize.NoViewport</c> to disable the default viewport.
+        /// Emulates consistent viewport for each page. Defaults to an 1280x720 viewport. <c>null</c>
```

Comment:
Upstream docs aren't right for this yet, either my PR gets merged, or we'll need to create a separate one with a separate field of `ignoreViewport` or something similar - which I'm strongly against. Either way, let's wait for that PR decision, and re-run this generator.

Reply:
Let's wait